Makefile

```diff
@@ -1,7 +1,27 @@
-name := pr/uk-core-functional-tests
+service := pr/uk-core-functional-tests
 repo := europe-west4-docker.pkg.dev/sbat-gcr-develop/sapig-docker-artifact
-tag  := latest
+tests := test
 
 docker:
-	docker build -t ${repo}/securebanking/${name}:${tag} .
-	docker push ${repo}/securebanking/${name}:${tag}
+ifndef tag
+	$(warning no tag supplied; latest assumed)
+	$(eval TAG=latest)
+else
+	$(eval TAG=$(shell echo $(tag) | tr A-Z a-z))
+endif
+ifndef setlatest
+	$(warning no setlatest true|false supplied; false assumed)
+	$(eval setlatest=false)
+endif
+	@if [ "${setlatest}" = "true" ]; then \
+		docker build -t ${repo}/securebanking/${service}:${TAG} -t ${repo}/securebanking/${service}:latest . ; \
+		docker push ${repo}/securebanking/${service} --all-tags; \
+    else \
+   		docker build  -t ${repo}/securebanking/${service}:${TAG} . ; \
+   		docker push ${repo}/securebanking/${service}:${TAG}; \
+   	fi;
+
+runTests:
+	@echo "Running tests suite '${tests}'"
+	sleep 5s
```

Comment:
Why do we need this?

Reply:
Added it into the OB functional tests a while back, makes running tests locally easier (with the profile it might make more sense) but it's not needed

Reply:
I'd just call it test, then you just type `make test`

Why do we need the sleep though?

Reply:
Will do

Ah that was to do with the profile and tests settings as well, so it would echo and you can check if its running against what you want before actually running, giving you chance to cancel

Reply:
haha, those damn profiles!